### ISSUE TITLE
Display roles assigned to a group

### ISSUE THREAD

**Issue by justinehell** [ditrit/leto-modelizer-admin#76]
Blocked by #84 

On GroupPage, add RolesTable to display all roles of a group.

Actions are not to implemented in this task.

![image](https://github.com/ditrit/leto-modelizer-admin/assets/62500081/8e81f028-a527-42bb-a190-a636e69f99f8)

**Comment by justinehell** [comment]
Task put on hold, will be done when the new api is up.
